frontend/src/components/crd/CustomResourceDetails.tsx

```diff
@@ -32,7 +33,7 @@ export function CustomResourceDetails(props: CustomResourceDetailsProps) {
   const { t } = useTranslation('glossary');
 
   const namespace = ns === '-' ? undefined : ns;
-  const CRD = ResourceClasses.CustomResourceDefinition as CustomResourceDefinition;
+  const CRD = ResourceClasses.CustomResourceDefinition;
```

Comment:
Can you try the type declaration instead (const CRD: CustomResourceDefinition = ...).

Reply:
casting was removed here, not added